bin/add_geneticmodels_to_svvcf.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+from pysam import VariantFile
+import cmdvcf
+import argparse
+from geneticmodels import GM
+
+def main(args: object):
+    """
+    Read VCF line by line, add genetic models to INFO-field
+    """
+    ped, individuals = read_ped(args.pedigree_file)
+    if len(ped.keys()) < 2:
+        exit("Cannot add genetic models to single samples")
```

Comment:
`sys.exit` I think (`exit` does not seem to be intended for production)

https://www.geeksforgeeks.org/python-exit-commands-quit-exit-sys-exit-and-os-_exit/